src/audio/drc/testbench_drc_asin/drc_asin_fixed.c

```diff
@@ -0,0 +1,115 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2021 Intel Corporation. All rights reserved.
+//
+// Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+/* Include Files */
+#pragma warning (disable : 4996)
+#pragma warning (disable : 4013)
+#include <stdio.h>
+#include <stdlib.h>
+#include "typedef.h"
+int32_t drc_asin_fixed(int32_t x, int8_t i, FILE* fd); /* Input:Q2.30 Output:Q2.30 */
+void init_struc_fixpt(int32_t x[21]);
+
+/*
+ * Input is Q2.30: (-2.0, 2.0)
+ * Output range: (-1.0, 1.0); regulated to Q1.31: (-1.0, 1.0)
+ */
+int main()
+{
```

Comment:
It's best practice to put main at the end and have all the static functions at the top of the file. This way you don't have to declare some functions at the start of the file.

Reply:
done